cmake/target/dict.cmake

```diff
@@ -81,7 +85,7 @@ function(add_module_target MODULE_NAME TARGET_NAME GLOBAL_TARGET_NAME AC_INPUTS
         # Only generate dictionaries on serializables or topologies
         if (AC_IN MATCHES ".*Topology.*\.xml$")
             fprime_ai_info("${AC_IN}" "${MODULE_NAME}")
-            dictgen("${MODULE_NAME}" "${AC_IN}" "${MODULE_DEPENDENCIES};${MOD_DEPS};${FILE_DEPENDENCIES}")
+            dictgen("${MODULE_NAME}" "${AC_IN}" "${AC_OUTPUTS}")
```

Comment:
Don't you also need to depend on on the module dependencies?  It seems just depending on AC_OUTPUTS will not detect changes in constituent components.

Reply:
You're right, I thought since the included xml files weren't compiled I didn't need to depend on them, but I forgot about triggering rebuilds. I'll fix that.